code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -496,6 +496,20 @@ This is here to make the tiles around the station mininuke change when it's arme
 	poi_list |= src
 	START_PROCESSING(SSobj, src)
 
+/obj/item/weapon/disk/nuclear/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/claymore/highlander))
```

Comment:
If it's really necessary, I'll do it, but I disagree. Putting it in the sword's `afterattack` is really slow, because you'll be swinging them a lot more often than you'll be hitting the nuke disk. That's why I put it in `attackby` - it'll trigger a lot less often and be much faster as a result.